### ISSUE TITLE
wazuh_agent_authd.agent_name defined as 'null' 

### ISSUE THREAD

**Issue by ketsapiwiq** [wazuh/wazuh-ansible#441]
Hi again!
So I had this problem with wazuh on windows: shouldn't it be [Jinja2's `is not none`](https://jinja.palletsprojects.com/en/2.11.x/templates/#none) here?
We find this in defaults/main.yml:
```yaml
wazuh_agent_authd:
      agent_name: null
```
But then in `Windows.yml` Jinja2 makes the template echo an empty string if the variable is `null` (apparently `none` in jinja):
```jinja2
    {% if wazuh_agent_authd.agent_name is defined %}-A {{ wazuh_agent_authd.agent_name }} {% endif %}
```
Which results in the failure that `-A` needs an argument (sorry I lost the output).

Or I missed yet another thing in Ansible variables.

I have a possible PR here if it's actually a bugfix: https://github.com/wazuh/wazuh-ansible/compare/master...ketsapiwiq:aca45e7d0078a9bf6453b1750f125230c1e7b2f5 

Thanks a lot!

**Comment by xr09** [comment]
Hey @ketsapiwiq, I reviewed the issue and you're right, I'm opening a PR with your commit.

Thanks for your contribution!

**Comment by ketsapiwiq** [comment]
Thank you!

**Comment by manuasir** [comment]
Merged here #442 thanks @ketsapiwiq
